src/core/message-parsing/directives/tool-directives/ToolParamName.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * List of parameter names that can be used in tool directives.
+ */
+export const toolParamNames = [
```

Comment:
Not too long ago Roo's code team moved all the tool logic into their own files, and I think we would prefer to continue that trend so your parser can operate without any tool-specific implementations:

How could you set this up so that each individual tool file to register supported parameters for that specific tool? 

This would isolate concerns and avoids this blob of tool-parameters that are comingled with other tools.